src/AllLayers.js

```diff
@@ -127,8 +128,16 @@ L.LayerGroup.environmentalLayers = L.LayerGroup.extend(
 
            L.control.layers(baseMaps,this.overlayMaps).addTo(map);
 
-		   if(this.options.hash)
-		     var hash = new L.Hash(map, this.overlayMaps);
+		    if(this.options.hash)
+				var hash = new L.Hash(map, this.overlayMaps);
```

Comment:
This looks great. Do you think we could do some formatting tidying up here, like standardized 2 spaces for indents, and `if (...)` with {} brackets for clarity? Thanks!!

This looks awesome! 🎉

Reply:
Thanks! Yes, there's some formatting required. Thought I could could tidy up things in a seperate issue so that I can go through all the files. What do you think?